### DISCUSSION TITLE
How do you have multiple installs of Directus 9 on one server?

### DISCUSSION THREAD

**Discussion by TomS-** [directus/directus#8973] [Archive]
I'm running into a host of problems on my second install of Directus. Much like PHP CMS's (WordPress, ProcessWire etc.) installing multiple instances is as simple as connecting it to a new database. 

However in Directus I'm finding that there is a strange symbolic link. Website 1 is running on port 8055, website 2 is running on port 8056. Both are on a domain with a reverse proxy to that port.

Website 1 works correctly, however website 2 refuses to keep my logged in. Each refresh will take me back to the login screen and I'm consistently faced with `[INVALID_TOKEN] Token expired.`. When this notice appears it also appears on website 1. 

Refreshing website 1 keeps me logged in even though it displayed the notice that the token expired while website 2 will always throw me back to the login screen. 

It's worth noting these are on completely separate databases, directories and ports with their own unique key and secret's.

I have tried renaming the token cookie name for website 2 but that has no effect.

Does Directus not support multiple installs on one server outside of Docker?

**Comment by TomS-** [comment]
One install was running RC.97 and another running RC.98, having them both on the same Directus version seemed to fix the problem.

**Reply by joselcvarela** [reply]
Hello @TomS- 
Yeah, I think you are running into this issue: https://github.com/directus/directus/issues/8947
It seems to be solved and will be released on rc99
